src/Request/StreamExtensions.cs

```diff
@@ -23,6 +28,13 @@ public static string AsString(this Stream stream, Encoding encoding = null)
             }
         }
 
+        /// <summary>
```

Comment:
Gets the <see cref="HttpRequest" /> Body <see cref="Stream"/> as <see cref="String"/> asynchronously in the optional <see cref="Encoding"/>